frontend/src/components/MetricsSummary.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Trans } from 'react-i18next';
+import css from './MetricsSummary.module.scss';
+
+const classes = classNames.bind(css);
+
+export const MetricsSummary: React.FC<{
+  label: string;
+  value: string | number;
+  children?: any;
+}> = ({ label, value, children }) => (
+  <div className={classes(css.dataEntryWrapper)}>
+    <Trans i18nKey={label} />
```

Comment:
I'm not quite sure if the label should be translated by the user of this component.
That might lead to less possible confusion

Reply:
labels are now translated by the user, not the component itself 👍